cpp/tests/test_abm.h

```diff
@@ -87,4 +83,30 @@ struct ScopedMockDistribution {
     typename Distribution::GeneratorFunction old;
 };
 
+/**
+ * @brief Add an Infection to the Person with the demanded InfectionState at the given time. Intended for simple use in tests.
+*/
+void add_infection_simple(mio::abm::Person& p,
+                          mio::abm::InfectionState infection_state   = mio::abm::InfectionState::Infected,
+                          mio::abm::TimePoint t                      = mio::abm::TimePoint(0),
+                          mio::abm::GlobalInfectionParameters params = {});
+
+/**
+ * @brief Create a Person without a World object. Intended for simple use in tests.
+*/
+mio::abm::Person create_person_simple(mio::abm::Location& location,
```

Comment:
I would propose new names, `make_test_person`, etc. highlights the purpose a bit better. Not really simple functions anyway!